ArduCopter/config.h

```diff
@@ -762,3 +762,7 @@
 #ifndef HAL_FRAME_TYPE_DEFAULT
 #define HAL_FRAME_TYPE_DEFAULT AP_Motors::MOTOR_FRAME_TYPE_X
 #endif
+
+#ifndef LANDING_GEAR_ENABLED
+ #define LANDING_GEAR_ENABLED !HAL_MINIMIZE_FEATURES
```

Comment:
this is wrong as we always have landing gear lib in copter.

Reply:
I was modeling off Plane for this. Dropped this change.